Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2463,7 +2463,7 @@ def lhegensim2018ml(fragment,howMuch):
                         '--datatier':'ALCARECO',
                         '--eventcontent':'ALCARECO',
                         '--conditions':'auto:run2_data',
-                        '--triggerResultsProcess':'RECO',
+                        '--triggerResultsProcess':'ALCARECO',
```

Comment:
@mmusich can you explain, just for my understanding what is the difference here between ALCARECO and RECO ?

Reply:
@grzanka none, it's a convention, but you need to keep it in synch with the one you used in the step before:
https://github.com/cms-sw/cmssw/blob/8391245a8a5febd6fdaec1e66395b67f9d2dd3db/Configuration/PyReleaseValidation/python/relval_steps.py#L2407

Reply:
Thanks for explanation. Does it mean that following this convention makes this PR to fix the reported issue ?